bin/build-plugin-zip.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+SOURCE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../" && pwd )"
+TMP_BUILD_DIR=/tmp/gutenberg
+
+# Build the assets
+npm install
+npm run build
+
+# Prepare the plugin directory
+rm -rf $TMP_BUILD_DIR
+mkdir -p $TMP_BUILD_DIR/i18n/build
+mkdir -p $TMP_BUILD_DIR/blocks/build
+mkdir -p $TMP_BUILD_DIR/editor/build
+mkdir -p $TMP_BUILD_DIR/element/build
+
+cp $SOURCE_DIR/README.md $TMP_BUILD_DIR
+cp $SOURCE_DIR/index.php $TMP_BUILD_DIR
+cp $SOURCE_DIR/post-content.js $TMP_BUILD_DIR
+cp -aR $SOURCE_DIR/i18n/build/* $TMP_BUILD_DIR/i18n/build
+cp -aR $SOURCE_DIR/blocks/build/* $TMP_BUILD_DIR/blocks/build
+cp -aR $SOURCE_DIR/editor/build/* $TMP_BUILD_DIR/editor/build
+cp -aR $SOURCE_DIR/element/build/* $TMP_BUILD_DIR/element/build
+
+# Zip the plugin
+rm -f $SOURCE_DIR/gutenberg.zip
+cd $TMP_BUILD_DIR; zip -r $SOURCE_DIR/gutenberg.zip *
```

Comment:
I think without the `cd`, it zips the folder name as well. I want the content only